compute/instances/how-to/migrate-routed-ips.mdx

```diff
@@ -13,15 +13,15 @@ categories:
   - compute
 ---
 
-During deployment, each Instance gets a private IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the private IP changes. This can happen when you restart your Instance. The private IP lets Instances communicate via the underlying private network, without public IPs. 
+During deployment, each Instance gets a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via the underlying private network, without public IPs. 
```

Comment:
```suggestion
During deployment, each Instance gets a NAT IP from its hosting node, besides its public IP, which is routed to the Instance through a [carrier-grade NAT](/compute/instances/concepts/#carrier-grade-nat-(cgnat)) solution. However, if the underlying hypervisor changes, the NAT IP changes. This can happen when you restart your Instance. The NAT IP lets Instances communicate via an underlying internal network, without public IPs. 
```